policy/modules/contrib/fdo.fc

```diff
@@ -5,6 +5,9 @@
 /etc/fdo/aio/logs(/.*)?					gen_context(system_u:object_r:fdo_conf_rw_t,s0)
 /etc/fdo/aio/stores(/.*)?				gen_context(system_u:object_r:fdo_conf_rw_t,s0)
 
+/etc/fdo/stores/manufacturing_sessions(/.*)?		gen_context(system_u:object_r:fdo_conf_rw_t,s0)
+/etc/fdo/stores/rendezvous_sessions(/.*)?		gen_context(system_u:object_r:fdo_conf_rw_t,s0)
```

Comment:
so, the fdo manufacturing-client that runs in the initrd, writes the file `/etc/device-credentials` but i can't see it anywhere here in the policy - can we add it and have another test package?